### PR TITLE
Fix bug the legend keys were in different order

### PR DESCRIPTION
**Pull request by lmichalevich**
Fixes https://github.com/demisto/etc/issues/37382

**screenshots**

Screen Shot 2021-05-30 at 13 36 03
Screen Shot 2021-05-30 at 13 36 14

### CODE REVIEW

File: src/components/Sections/SectionChart/SectionBarChart.js [hunk 1]
```diff
@@ -193,7 +193,12 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
       }
       return dataItem;
     });
+
+    if (stacked) {
```
Comment (yanyo87) [inline src/components/Sections/SectionChart/SectionBarChart.js:197]: why we cannot leave it in the original place ?

**Review by yanyo87 — COMMENTED** [review]
Let's add a spec so we will not miss it again . 
or fix the existing spec

File: src/components/Sections/SectionChart/SectionBarChart.js (hunk 1 above)
Comment (lmichalevich) [reply src/components/Sections/SectionChart/SectionBarChart.js:197]: the code inside the if(legend) modify the order

Comment (yanyo87) [reply src/components/Sections/SectionChart/SectionBarChart.js:197]: the code inside the if shouldn't change the order

Comment (lmichalevich) [reply src/components/Sections/SectionChart/SectionBarChart.js:197]: this is the equivalent for the code in the web-client:
```
allSubGroups = Object.values(allSubGroups).sort((a, b) => sortByField(['sum', 'name'], [false, true])(a, b));
```

**Review by yanyo87 — APPROVED** [review]
@lmichalevich Looks good. 

@Bars92 can you have a quick look as well ?

File: src/utils/validators.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import validator from 'validator';
+
+export function isNumber(value) {
+  return Number.isInteger(value) || (value && validator.isNumeric(value));
```
Comment (Bars92) [inline src/utils/validators.js:4]: why do you need `validator` here? isNaN should suffice no?

Comment (lmichalevich) [reply src/utils/validators.js:4]: This is the code from the web-client project (this code was added because a pr comment)

**Pull request merged**